astropy/units/deprecated.py

```diff
@@ -41,6 +47,7 @@ def _initialize_module():
 # This generates a docstring for this module that describes all of the
 # standard units defined here.
 from .utils import generate_unit_summary as _generate_unit_summary  # noqa
+from .utils import generate_prefixonly_unit_summary as _generate_prefixonly_unit_summary  # noqa
```

Comment:
I just copied it from the line above.  I actually don't know why it was in the *previous* line - I'm guessing at least someone's PEP8 or similar checker doesn't like the `as` style or something?  I figured safer to be consistent with the line just above.